src/Entity/Field.php

```diff
@@ -200,11 +200,11 @@ public function getFlattenedValue()
     }
 
     /**
-     * @return string|Markup
+     * @return string|array|Markup
      */
     public function getTwigValue()
     {
-        $value = $this->__toString();
+        $value = $this->getFlattenedValue();
 
         if ($this->getDefinition()->get('allow_html')) {
```

Comment:
`if (is_string($value) && $this->getDefinition()->get('allow_html')) {`

Reply:
or Markup allows arrays too?

Reply:
No, `Markup` is always a `string` with the contents, plus a `string` with the encoding.

Reply:
Ok but using allow_html on i.e. image field doesn't make sense and it will break here if it happens, no?

Reply:
Ah, yes, you are correct. Fixed.